python/tvm/relay/grammar/Relay.g4

```diff
@@ -17,7 +17,11 @@
  * under the License.
  */
 
-// list = *, seq = ?
+// TODO(weberlo): We need some way of indicating to users that you need to enable
```

Comment:
Is a comment in this file (and one in the Relay docs) reminding users that the parser will not be updated unless the flag is set not sufficient?

Reply:
Yeah.  It probably is sufficient.  I was thinking more along the lines of a programmatic enforcement, but the number of people working on this area of the codebase doesn't really warrant that.